src/App.css

```diff
@@ -96,3 +93,47 @@
   padding-left: 5px;
   color: #060553;
 }
+
+.user_info_container {
+  line-height: 60px;
+  float: right;
+  font-weight: 500;
+}
+
+.dashboard_link {
+  color: #060553;
+  text-decoration-color: #00d4ff;
+  text-decoration-thickness: 2px;
+  margin-right: 30px;
+}
+
+.edit_profile_link {
+  color: #060553;
+  text-decoration-color: #00d4ff;
+  text-decoration-thickness: 2px;
+  margin-right: 90px;
+}
+
+.about_container {
+  height: 610px;
+  color: #e5e5e5;
+  background: linear-gradient(
+    360deg,
+    #020024 11.39%,
+    rgba(8, 8, 115, 0.709638) 93.3%,
+    rgba(9, 9, 121, 0.685897) 100%
+  );
+}
+
+.about_container h1 {
+  position: absolute;
+  left: 3%;
```

Comment:
I suppose this is the style from Figma, no? The `h1` element does not need to be positioned absolutely.

Reply:
Yes, it's the style from figma. However, removing the absolute position changes the position of the text and also does this stuff where it reduces the background height as well
![Screenshot from 2020-09-23 17-22-35](https://user-images.githubusercontent.com/18626169/94040957-829c0580-fdc1-11ea-9483-07a12c4bb043.png)

Reply:
@Nkoli you can use margins and paddings to achieve the same styling. Absolute positioning is great, but it is easy for it to get out of hand, so my approach is to use `position: absolute` for things like `Modals`, `Popovers`, `Tooltips`, `Dropdowns`, `DatePickers` etc

Reply:
Okay, I'll use margins and paddings for this. I thought using absolute and then defining the percentage value for specific positioning allowed for better responsiveness than using margin and paddings for components that may be reused in different ways?